pkgs/by-name/go/gophertube/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  yt-dlp,
+  mpv,
+  fzf,
+  chafa,
+  makeBinaryWrapper,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "gophertube";
+  version = "2.7.0";
+
+  src = fetchFromGitHub {
+    owner = "KrishnaSSH";
+    repo = "GopherTube";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-xsEK+iSobGOxiCx9lkQcxm6SIplTTbnXwP5/RzJQF+Q=";
+  };
+
+  vendorHash = "sha256-WfVoCxzMk+h4AP1zgTNRXTpj8Ltu71YrsQ7OoU3Y4tg=";
+
```

```suggestion

  ldflags = [
    "-X gophertube/internal/app.version=${finalAttrs.version}"
  ];

```

Currently `gophertube --version` returns `GopherTube version dev`
With this change, it returns `GopherTube version 2.7.0`